TaylorFrameworkTests/CapriceTests/Tests/FlagsTests.swift

```diff
@@ -10,6 +10,15 @@ import Quick
 import Nimble
 @testable import TaylorFramework
 
+class HelpFlagMock: HelpFlag {
+    func executeTest() -> Bool {
```

Comment:
Could we move this method in a helper class in Tests bundle?  
It should be not to modify the production code just because we need to test it were possible.